armips/data/moves.s

```diff
@@ -8999,7 +8999,7 @@ movedata MOVE_FUSION_FLARE, "Fusion Flare"
     appeal 0x00
     contesttype CONTEST_COOL
     terminatedata
-    movedescription MOVE_FUSION_FLARE, "The user brings down\na huge flame. It does\nmore damage when\ninfluenced by an\nenormous bolt."
+    movedescription MOVE_FUSION_FLARE, "The user throws down\na giant flame.\nIt does more damage\nwhen influenced by\nan enormous bolt."
```

Comment:
This is not correct.

[https://bulbapedia.bulbagarden.net/wiki/Fusion_Flare_(move)#Description](https://bulbapedia.bulbagarden.net/wiki/Fusion_Flare_(move)#Description)